robosherlock/src/queryanswering/include/rs/queryanswering/RSSimpleQueryActionServer.h

```diff
@@ -0,0 +1,70 @@
+#include <robosherlock_msgs/RSSimpleQueryAction.h>  // Note: "Action" is appended
+#include <actionlib/server/simple_action_server.h>
+#include <rs/flowcontrol/RSProcessManager.h>
+#include <sstream>
+#include <string.h>
+#include <std_msgs/String.h>
+#include <vector>
+#include <iostream>
+
+
+
+ typedef actionlib::SimpleActionServer<robosherlock_msgs::RSSimpleQueryAction> Server;
+
+
+class RSSimpleQueryActionServer{
+
+private:
+
+    /*
+     * Attributes
+     */
+    Server* server;
+    RSProcessManager* rsProcessManager;
```

Comment:
I would do it the other way around. Mainly because of future plans. Namely, RSSimpleQueryActionServer should be a member of the RSProcessManager class;

Reply:
done